docs/proposal/adsmod.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Kmesh ads mode proposal
+authors:
+- "@LiZhenCheng9527"
+reviewers:
+approvers:
+
+creation-date: 2024-04-28
+
+---
+
+## Backgroud
+
+Because Kmesh provides a new data plane architecture, the control plane still uses istiod. So still need `ads`` for Kmesh and Istiod synchronisation.
+
+## Kmesh ads mode Function Design
+
+The process of synchronising Kmesh with istiod is the same as synchronising envoy with istiod:
+
+- **Create grpc links with istiod.**
+  After creating the grpc link, use the `streamAggregatedResources`` method to establish a link with istiod.
+- **Getting the initial configuration of the cluster.**
+  After establishing a link, Kmesh sends a clusterType empty ads message to istiod to get a snapshot of the current cluster configuration resources.
```

Comment:
I think you mean kmesh send the initial CDS subscription request, so please organize this sentence well